addons/resource/views/resource_views.xml

```diff
@@ -291,15 +293,6 @@
                         </group>
                             <field name="attendance_ids" widget="section_one2many"/>
                         </page>
-                        <page string="Global Time Off" name="global_leaves">
```

Comment:
Why do we remove them ? 

I would at least hide them in debug mode for debugging purpose.